scom/stackstate_checks/scom/scom.py

```diff
@@ -55,7 +56,24 @@ def get_instance_key(self, instance):
 
         instance_url = instance['hostip']
         return TopologyInstance(self.INSTANCE_TYPE, instance_url)
-
+    def sendAlertsPowershell(self,scom_server):
+        cmd = subprocess.Popen(["powershell.exe","Start-OperationsManagerClientShell -managementServerName "+scom_server+"\n","Get-SCOMAlert | select \"monitoringobjectid\", \"name\", \"monitoringobjectdisplayname\", \"description\", \"resolutionstate\", \"monitoringobjectpath\" -ExpandProperty  timeadded| ConvertTo-Json \n"],stdout=subprocess.PIPE)
+        out = cmd.communicate()[0]
+        alerts = json.loads(out)
```

Comment:
In other checks `yaml.loads` is used. You should also be aware of what Unicode encoding is used. Any idea?

Reply:
yaml.loads with json ? 
json.loads converts a string to json object. the string is for sure json because i convert it in powershell
powershell output is ASCII but can be set to utf-8 for example